woocommerce/woocommerce-gutenberg-products-block
# Product Collection: Add 'Order By' Control to Product Collection Inspector

Add 'Order By' Control to Product Collection Inspector.

<img width="289" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/011b91ec-868c-4c21-a91f-b8a02d025c62">


This PR adds a new 'Order By' control to the product collection inspector. The control allows users to specify the order of products by various attributes such as title and date. 

Fixes #9359 




### Testing

1. Create a new post
2. Add Product Collection block
3. Verify that the sidebar displays a 'Order By' setting when the Product Collection block is selected.
4. Change the order using the 'Order By' setting.
   - Confirm that the change is immediately reflected in the editor.
5. Save and publish the post.
   - Verify that the same change is reflected on the frontend.


### WooCommerce Visibility




### Changelog

> Product Collection: Add 'Order By' Control to Product Collection Inspector
